### PR TITLE
Updated Assigning Procedure

### PR DESCRIPTION
**Pull request by Exnav29**
Updated procedures to assign eDiscovery Managers and eDiscovery Administrators.  The updates are in line with the wording used in the UI.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/assign-ediscovery-permissions.md [hunk 1]
```diff
@@ -53,11 +53,11 @@ The primary eDiscovery-related role group in Security & Compliance Center is cal
...
-    
-  - **To make a user an eDiscovery Administrator** Next to **eDiscovery Administrator**, click **Edit**. Under **Selected eDiscovery Administrators**, click **Edit**, and then click ![Add Icon](media/ITPro-EAC-AddIcon.gif) **Add**. Select the user (or users) you want to add as an eDiscovery Administrator, and then click **Add**. When you're finished adding users, click **Done**. Then, on the **Editing Choose eDiscovery Administrator** flyout page, click **Save** to save the changes to the eDiscovery Administrator membership. 
-    
+  
+  **To make a user an eDiscovery Manager** Next to **eDiscovery Manager**, click **Edit**. In the **Choose eDiscovery Manager** section, click the **Choose eDiscovery Manager** hyperlink and then click ![Add Icon](media/ITPro-EAC-AddIcon.gif) **Add**. Select the user (or users) you want to add as an eDiscovery manager, and then click **Add**. When you're finished adding users, click **Done**. Then, on the **Editing Choose eDiscovery Manager** flyout page, click **Save** to save the changes to the eDiscovery Manager membership.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/assign-ediscovery-permissions.md]:
```suggestion
  **To make a user an eDiscovery Manager**: Next to **eDiscovery Manager**, click **Edit**. In the **Choose eDiscovery Manager** section, click the **Choose eDiscovery Manager** hyperlink and then click ![Add Icon](media/ITPro-EAC-AddIcon.gif) **Add**. Select the user (or users) you want to add as an eDiscovery manager, and then click **Add**. When you're finished adding users, click **Done**. Then, on the **Editing Choose eDiscovery Manager** flyout page, click **Save** to save the changes to the eDiscovery Manager membership.
```

File: microsoft-365/compliance/assign-ediscovery-permissions.md [hunk 2]
```diff
@@ -53,11 +53,11 @@ The primary eDiscovery-related role group in Security & Compliance Center is cal
...
+  
+  **To make a user an eDiscovery Administrator** Next to **eDiscovery Manager**, click **Edit**. In the **Choose eDiscovery Administrator** section, Under **eDiscovery Administrators**, click **Choose eDiscovery Administrator** click **Edit**, and then click ![Add Icon](media/ITPro-EAC-AddIcon.gif) **Add**. Select the user (or users) you want to add as an **eDiscovery Administrator**, and then click **Add**. When you're finished adding users, click **Done**. Then, on the **Editing Choose eDiscovery Administrator** flyout page, click **Save** to save the changes to the eDiscovery Administrator membership.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/assign-ediscovery-permissions.md]:
```suggestion
  **To make a user an eDiscovery Administrator**: Next to **eDiscovery Manager**, click **Edit**. In the **Choose eDiscovery Administrator** section, under **eDiscovery Administrators**, click **Choose eDiscovery Administrator**. Click **Edit**, and then click ![Add Icon](media/ITPro-EAC-AddIcon.gif) **Add**. Select the user (or users) you want to add as an **eDiscovery Administrator**, and then click **Add**. When you're finished adding users, click **Done**. Then, on the **Editing Choose eDiscovery Administrator** flyout page, click **Save** to save the changes to the eDiscovery Administrator membership.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Pull request merged**